liberty-maven-plugin/src/it/tests/appsdirectory-set-appz-configured-it/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>net.wasdev.wlp.maven.it</groupId>
+    <artifactId>tests</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>appsdirectory-set-appz-configured-it</artifactId>
+  <packaging>liberty-assembly</packaging>
+
+  <dependencies>
+    <dependency>
+        <groupId>org.apache.geronimo.specs</groupId>
+        <artifactId>geronimo-servlet_3.0_spec</artifactId>
+        <version>1.0</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>commons-logging</groupId>
+        <artifactId>commons-logging</artifactId>
+        <version>1.0.4</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.9</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>net.wasdev.wlp.maven.plugins</groupId>
+        <artifactId>liberty-maven-plugin</artifactId>
+        <version>@pom.version@</version>
+        <extensions>true</extensions>
+        <configuration>
+          <stripVersion>true</stripVersion>
+          <assemblyArtifact>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>assembly-server</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+          </assemblyArtifact>
+          <serverName>test</serverName>
+          <installAppPackages>project</installAppPackages>
+          <appsDirectory>appz</appsDirectory>
+          <stripVersion>true</stripVersion>
+          <skipTestServer>true</skipTestServer>
+          <looseApplication>true</looseApplication>
+        </configuration>
+        </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-war-plugin</artifactId>
+            <version>2.3</version>
```

Comment:
Actually, so you even need this plugin since you are not building the war and using looseApplication? Originally, I thought it was included to configure <failOnMissingWebXml>false</failOnMissingWebXml>, but it doesn't appear to be configured here.